src/quo2/components/inputs/input/style.cljs

```diff
@@ -61,13 +61,15 @@
        :text         (colors/theme-colors colors/neutral-100 colors/white theme)})))
 
 (defn input-container
-  [colors-by-status small? disabled?]
-  {:flex-direction     :row
-   :padding-horizontal 8
-   :border-width       1
-   :border-color       (:border-color colors-by-status)
-   :border-radius      (if small? 10 12)
-   :opacity            (if disabled? 0.3 1)})
+  [container-style colors-by-status small? disabled?]
+  (assoc container-style
```

Comment:
one small issue with this is that we want the priority of containers-style to be greater so it can override these styles if necessary. is that alright with assoc or it's better to use `(merge {...}  container-style)` ?

Reply:
@J-Son89 Now the container-style is being applied in a different node so that we are prioritizing it.

I had to continuosly adapt this code. It was changed originally in df2b5147cd717f2a7d88549230fcbe19e7ea62c7 and then partially reverted in 1812cfb8dc9e6cf5e5849e93a66550a6d35dc20e. I hope now this is stable.